src/sentry/search/events/fields.py

```diff
@@ -112,6 +116,80 @@ def key_transaction_expression(user_id, organization_id, project_ids):
     ]
 
 
+def project_threshold_config_expression(organization_id, project_ids):
```

Comment:
Could you add a summary of the what the query does here at a high level? The json is pretty complex so future readers will need all the help they can get.